build.gradle

```diff
@@ -4,52 +4,76 @@ buildscript {
     repositories {
         mavenLocal()
         mavenCentral()
-        maven { url "http://repo.spring.io/plugins-release" }
-        maven { url "https://plugins.gradle.org/m2/" }
+        gradlePluginPortal()
+        maven { url "https://repo.spring.io/plugins-release" }
     }
     dependencies {
-        classpath "org.springframework.boot:spring-boot-gradle-plugin:${spring_boot_version}"
-        classpath "io.spring.gradle:propdeps-plugin:0.0.10.RELEASE"
-        classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:1.5.2"
         //jhipster-needle-gradle-buildscript-dependency - JHipster will add additional gradle build script plugins here
     }
 }
 
 plugins {
-    id "org.sonarqube" version "2.6.2"
-    id "net.ltgt.apt-eclipse" version "0.19"
-    id "net.ltgt.apt-idea" version "0.19"
-    id "net.ltgt.apt" version "0.19"
-    id "io.spring.dependency-management" version "1.0.6.RELEASE"
-    id 'org.liquibase.gradle' version '2.0.1'
-    id "com.github.spotbugs" version "1.6.3"
+    id "java"
+    id "maven-publish"
+    id "idea"
+    id "jacoco"
+    id "org.springframework.boot"
+    id "com.google.cloud.tools.jib"
+    id "com.gorylenko.gradle-git-properties"
+    id "net.ltgt.apt-eclipse"
+    id "net.ltgt.apt-idea"
+    id "net.ltgt.apt"
+    id "org.liquibase.gradle"
+    id "org.sonarqube"
+    id "io.spring.nohttp"
+    id "com.github.spotbugs"
     //jhipster-needle-gradle-plugins - JHipster will add additional gradle plugins here
 }
 
-spotbugs {
-    toolVersion = '3.1.3'
-    ignoreFailures = true
-}
+group = "com.icthh.xm.ms.scheduler"
+version = "0.0.1-SNAPSHOT"
 
-apply plugin: 'java'
-sourceCompatibility=1.8
-targetCompatibility=1.8
-// Until JHipster supports JDK 9
-assert System.properties['java.specification.version'] == '1.8'
+description = ""
 
-apply plugin: 'maven'
-apply plugin: 'org.springframework.boot'
+sourceCompatibility=11
+targetCompatibility=11
+assert System.properties["java.specification.version"] == "1.8" || "11" || "12" || "13"
+
+apply from: "gradle/docker.gradle"
+apply from: "gradle/sonar.gradle"
 apply plugin: 'war'
-apply plugin: 'propdeps'
-apply plugin: 'io.spring.dependency-management'
-apply plugin: 'idea'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
+apply plugin: 'com.github.spotbugs'
```

Comment:
apply plugin it's a old style, better to use id "war" in plugins section

Reply:
done